public/index.html

```diff
@@ -5,7 +5,19 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,500|Roboto:400,500" rel="stylesheet">
-    <title>Blockchain Explorer</title>
+    <title>æternity explorer</title>
+
+    <meta property="og:title" content="æternity explorer">
+    <meta property="og:site_name" content="æternity explorer">
+    <meta property="og:url" content="https://explorer.aepps.com">
+    <meta property="og:description" content="æternity blockchain explorer shows detailed information on blocks and transactions.">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="/explorer-open-graph.png">
```

Comment:
@ror-shubham same thing here for the image, it needs the `BASE_URL` but also check if the `BASE_URL` is the correct one that points to the assets.